ports/psoc6/boards/CY8CPROTO-062-4343W/bsps/TARGET_APP_CY8CPROTO-062-4343W/COMPONENT_CM4/TOOLCHAIN_GCC_ARM/linker.ld

```diff
@@ -86,7 +86,7 @@ MEMORY
     sflash_public_key (rx)    : ORIGIN = 0x16005A00, LENGTH = 0xC00        /* Supervisory flash: Public Key */
     sflash_toc_2      (rx)    : ORIGIN = 0x16007C00, LENGTH = 0x200        /* Supervisory flash: Table of Content # 2 */
     sflash_rtoc_2     (rx)    : ORIGIN = 0x16007E00, LENGTH = 0x200        /* Supervisory flash: Table of Content # 2 Copy */
-    xip               (rx)    : ORIGIN = 0x18000000, LENGTH = 0x8000000    /* 128 MB */
+    xip               (rx)    : ORIGIN = 0x18000000, LENGTH = 0x4000000    /* 512 Mb / 64 MB */
```

Comment:
I just saw that XIP is not available for QSPI. Or am I mistaken ? In that case the change here is not required, right ? Because we may change the structure of the boards/<BOARD> directories in the future, this file should go directly to boards/CY8CPROTO-062-4343W if really required.

Reply:
The `XIP section` is present in the linker scripts for all the MTB examples. But nothing is populated in this section except for the dedicated `QSPI XIP application example`. 

In that available linker script, I have just changed the capacity value to reflect what's true for the CYPROTO 4343 board (XIP region would be 512 Mb). I agree that since every board has its own configuration with different QSPI FLASH chips present, every board must get its own linker script. Could we keep this version for now since we are using the CYPROTO 4343 board for now?

Reply:
We can keep it for now, but in case the BSP is updated the changes will be lost. That is uncritical as the part is not used. So we should make this part of the doc and undo the changes and thus be back on the base BSP. Any real change will later be reflected by another BSP or a borad specific linker script, as you said.